OSDOCS-17075_c#Updating clusters CQA (#115160)
openshift/openshift-docs: modules/manually-maintained-credentials-upgrade.adoc

[id="manually-maintained-credentials-upgrade_{context}"]
= Manually updating cloud provider resources

[role="_abstract"]

> Manually update cloud provider credentials by creating secrets for new components and adjusting permissions for existing components to meet the requirements of the target release.

Reviewer: ```suggestion
Meet the requirements of the target release by manually updating cloud provider credentials. Update these credentials by creating secrets for new components and by adjusting the permissions for existing components.
```
imho meeting the reqs for the release is the real task

.Procedure

. Create YAML files with secrets for any `CredentialsRequest` custom resources that the new release image adds. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object.

> .Sample AWS `CredentialsRequest` object with secrets

Reviewer: ```suggestion
+
--
.Sample AWS `CredentialsRequest` object with secrets
```
not aligned in preview, and must be attached to the step
attach all of these or they'll get lost; pretty sure you'll need an open block to get the numbering corrected

  aws_access_key_id: <base64_encoded_aws_access_key_id>
  aws_secret_access_key: <base64_encoded_aws_secret_access_key>
----

> 

Reviewer: ```suggestion
+
```

  azure_resourcegroup: <base64_encoded_azure_resourcegroup>
  azure_region: <base64_encoded_azure_region>
----

> 

Reviewer: ```suggestion
+
```
etc, attach it all together inside the block (i think this will work)

data:
  service_account.json: <base64_encoded_gcp_service_account_file>

> ----

Reviewer: ```suggestion
----
--
```
end open block

> . If the `CredentialsRequest` custom resources for any existing credentials that are stored in secrets have changed permissions requirements, update the permissions as required.

Reviewer: this renumbered as #1, so I think you'll need an open block for all of example hooked to the previous step
